acme/authority.go

```diff
@@ -47,11 +47,28 @@ type Interface interface {
 
 // Authority is the layer that handles all ACME interactions.
 type Authority struct {
+	backdate *provisioner.Duration
 	db       nosql.DB
 	dir      *directory
 	signAuth SignAuthority
 }
 
+// Options required to create a new ACME Authority.
+type AuthorityOptions struct {
+	Backdate *provisioner.Duration
```

Comment:
The `Backdate` is a pointer in `authority.AuthConfig` because we want the ability to be able to set it to `0` using the ca.json. By default, `nil`, we set it to one minute.

If you are using the `NewAuthority` programmatically outside of our CA and you do not set it, it will be null and it will panic. A better approach would be to use just `provisioner.Duration` that will be 0 by default.

Reply:
You might also make an alias as you did with TimeDuration, it will be easier to use.